src/helpers/tsReader.ts

```diff
@@ -372,7 +372,11 @@ export const registerUserTs = (basePath: string): (() => void) | null => {
 
   // handle path aliases
   const tsConfigString = fs.readFileSync(foundPath, "utf8");
-  const tsConfig = JSON.parse(stripJsonComments(tsConfigString));
+
+  const tsConfig = stripJsonComments(tsConfigString) as unknown as { [key: string]: any };
```

Comment:
This `JSON.parse` is required to parse the user's TSConfig file and register the `paths` option. The `stripJsonComments` returns a string, so this cannot simply be casted to `{ [key: string]: any };`.

Parsing a tsconfig.json file with `JSON.parse` should not throw errors unless the file is invalid. A better alternative in my opinion is to wrap the `JSON.parse` in a try/catch and give a specific error message, eg:

```typescript
try {
   const tsConfig = JSON.parse(stripJsonComments(tsConfigString));
} catch(err) {
  throw new Error("Error parsing your tsconfig.json file, please ensure the format is valid");
}
```

Reply:
Strange this behavior that you described is not happening when running this project currently when we install or clone it.

locally I needed to remove JSON.parser and everything worked.

I was curious wondering if it was my machine or operating system.

I did the same tests that I described in the issue and have different operating systems (macOs, ubuntu and windows 10), they all gave the same error. even using the example in the README.

today this lib runs locally in your environment without having this JSON.parser problem?

Reply:
Yep we've been using this lib in production at my company for a few months without issues. 

Are you sure your `tsconfig.json` file is correct? If so, then JSON.parse shouldn't have any issues. All the code is doing is reading this file and then parsing it. Can you try printing out the output from `stripJsonComments(tsConfigString)` and post here? Will help quite a bit